templates/cart/cart-empty.php

```diff
@@ -25,7 +25,7 @@
 if ( wc_get_page_id( 'shop' ) > 0 ) : ?>
 	<p class="return-to-shop">
 		<a class="button wc-backward" href="<?php echo esc_url( apply_filters( 'woocommerce_return_to_shop_redirect', wc_get_page_permalink( 'shop' ) ) ); ?>">
-			<?php esc_html_e( 'Return to shop', 'woocommerce' ); ?>
+			<?php echo apply_filters('woocommerce_return_to_shop_title', sprintf( '%s', __('Return To Shop', 'woocommerce' ) )); ?>
```

Comment:
Why there's a sprintf on this filter? I can't see any real work use for it.
Also not following WordPress coding standards.